ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestsPage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import {
+    Flex,
+    FlexItem,
+    PageSection,
+    Tab,
+    TabTitleText,
+    Tabs,
+    Title,
+} from '@patternfly/react-core';
+
+type TabKey = 'PENDING_APPROVALS' | 'APPROVED_DEFERRALS' | 'APPROVED_FALSE_POSITIVES';
+
+function ExceptionRequestsPage() {
+    const [activeTabKey, setActiveTabKey] = useState<TabKey>('PENDING_APPROVALS');
+
+    const handleTabClick = (event, tabIndex) => {
+        setActiveTabKey(tabIndex);
+    };
+
+    return (
+        <>
+            <PageSection
+                className="pf-u-display-flex pf-u-flex-direction-row pf-u-align-items-center"
+                variant="light"
+            >
+                <Flex direction={{ default: 'column' }}>
+                    <Title headingLevel="h1">Exception Management</Title>
```

Comment:
Small small nit:
```suggestion
                    <Title headingLevel="h1">Exception management</Title>
```

Reply:
Should we discuss as team and maybe also double check at an upcoming PatternFly meeting where is the boundary between Title Case in sidebar and Sentence case for the same thing as page title and heading.